frontend-react/src/styles/_uswds_settings.scss

```diff
@@ -64,6 +64,9 @@ $flex-wrap-settings: (
   responsive: true
 );
 
+// Add all USWDS colors: https://designsystem.digital.gov/design-tokens/color/system-tokens/
+$global-color-palettes: "palette-color";
+
```

Comment:
This adds all the needed colors from `https://designsystem.digital.gov/design-tokens/color/system-tokens/` that design uses across their Figma.